uint8 cam_type
uint8 num_result
BBox[] bboxes